components/google_drive/constants.mjs

```diff
@@ -157,6 +157,12 @@ const GOOGLE_DRIVE_GRANTEE_TYPES = [
   GOOGLE_DRIVE_GRANTEE_ANYONE,
```

Comment:
Move this file to `common/constants.mjs`

Reply:
Maybe we should do this in a different PR? If I move these files probably we will have to test all actions and sources again.